lntest/itest/lnd_test_list_on_test.go

```diff
@@ -3,6 +3,10 @@
 package itest
 
 var allTestCases = []*testCase{
+	{
+		name: "test multi-hop htlc",
+		test: testMultiHopHtlcClaims,
```

Comment:
What's the rationale for this change?

Reply:
Added a comment to the commit:

> To make sure the test that takes the longest overall time is always
started first, independent of the number of test tranches we run, we
move it to the beginning of the list. Because that test involves a lot
of waiting, it allows us to play around with the number of tranches more
efficiently.